isar/analysis_options.yaml

```diff
@@ -0,0 +1,33 @@
+include: package:lints/recommended.yaml
+
+analyzer:
+  language:
+    strict-casts: true
+    strict-inference: true
+    strict-raw-types: true
+  errors:
+    # treat missing required parameters as a warning (not a hint)
+    missing_required_param: error
+    # treat missing returns as a warning (not a hint)
+    missing_return: error
+    # allow having TODOs in the code
+    todo: ignore
+    # allow self-reference to deprecated members (we do this because otherwise we have
+    # to annotate every member in every test, assert, etc, when we deprecate something)
+    deprecated_member_use_from_same_package: warning
+    # ignore sentry/path on pubspec as we change it on deployment
+    invalid_dependency: ignore
+    unnecessary_import: ignore
```

Comment:
I know this is prolly copied from another package but do you know why we ignore these? I recently got flagged by the analyzer because `unnecessary_import` was ignored and didn't trigger in my code